### PR TITLE
Add profile page and url and basic tab component

### PR DESCRIPTION
**Pull request by Dee-McG**

### CODE REVIEW

File: dynamiccv/urls.py [hunk 1]
```diff
@@ -23,4 +23,5 @@
     path("ckeditor5/", include('django_ckeditor_5.urls')),
     path('djrichtextfield/', include('djrichtextfield.urls')),
     path('', include('home.urls')),
+    path('profile', include('profiles.urls')),
```
Comment (stephendawsondev) [inline dynamiccv/urls.py]: Issue (blocking): You just need the `/` at the end of `profile` so that it is `profile/` or else it doesn't seem to work.

**Review by stephendawsondev — CHANGES_REQUESTED** [review]
Looks good, just one minor change where the `/` is needed at the end of `profile` for the URL to work in the `dynamiccv/urls.py`.

File: dynamiccv/urls.py (hunk 1 above)
Comment (Dee-McG) [reply dynamiccv/urls.py]: I'll add it for the other routes, it currently works as we only have a base url view so there's no trailing text after, good catch

Comment (Dee-McG) [reply dynamiccv/urls.py]: Fixed

**Review by stephendawsondev — APPROVED** [review]
All good now! LGTM! 🚢

**Pull request merged**